draft/spec/index.html

```diff
@@ -698,6 +699,14 @@ <h2>Version</h2>
                             containing <a>logical path</a> names of files in the OCFL Object state that have
                             content with the given digest.
                         </p>
+                        <p>
+                            <a href="#logical-path">Logical paths</a> present the structure of an OCFL Object at a
+                            given version. This is given as an array of file paths, with the following restrictions
+                            provide for path safety in this common case. The logical path MUST be interpreted as a
```

Comment:
If I was being particularly bolshy, I'd say "The logical path MUST be interpreted as a set of one or more elements joined by a <code>/</code> separator. Elements MUST NOT be <code>.</code>, <code>..</code>, or empty (<code>//</code>)."

Reply:
At that point aren't we just obfuscating?

Reply:
I think the current wording is a bit clearer, even if it uses more words.